
max_frames_to_skip: 15
dist_threshold: 54.0
center_threshold: 80.0
area_threshold: 2.0
body_ratio: 0.5
dt: 0.02
use_z: true
use_dim: true
Q:
 - 3.0
 - 3.0
 - 0.3
 - 3.0
 - 3.0
 - 0.3
 - 5.0
 - 5.0
R:
 - 5.0
 - 5.0
 - 0.3
 - 5.0
 - 5.0
 - 0.3
 - 3.0
 - 5.0
 min_bbox_width: 60
 max_bbox_width: 400
 min_bbox_height: 60
 max_bbox_height: 300